staging/src/k8s.io/pod-security-admission/test/fixtures_windowsHostProcess.go

```diff
@@ -46,6 +46,8 @@ func init() {
 		failRequiresFeatures: []featuregate.Feature{"WindowsHostProcessContainers"},
```

Comment:
drop the changes to this file, the podsecurity check doesn't care if the OS is set or not

Reply:
I wanted this field to be explicit for windows pods but I removed it now.